### PR TITLE
Refactor/coremod to core

### PR DESCRIPTION
**Pull request by Raycoms**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Refactors the coremod package to core.
- Translations strings remain the same to avoid a mess for translators for now.
- Add a slight extra code to the RequestSystem Factory to rename coremod to core (typetokens need that).
- Rename Node from the miner to MineNode

[ x ] Yes I tested this before submitting it.
[ ] I also did a multiplayer test.

Review please

### CODE REVIEW

**Review by Thodor12 — DISMISSED** [review]
I'll just approve this, by the time I've managed to scroll through this it'd likely be midnight

**Review by someaddons — DISMISSED** [review]

File: src/api/java/com/minecolonies/api/colony/requestsystem/StandardFactoryController.java [hunk 1]
```diff
@@ -314,22 +307,11 @@ public <OUTPUT> OUTPUT deserialize(@NotNull final CompoundTag compound) throws I
         }
         catch (Throwable throwable)
         {
-            Log.getLogger().error(throwable);
+            Log.getLogger().error("Error when deserializing", throwable);
             return null;
         }
     }
 
-    private String processClassRenaming(@NotNull final String previousClassName)
```
Comment (MotionlessTrain) [inline src/api/java/com/minecolonies/api/colony/requestsystem/StandardFactoryController.java:322]: I guess this is an old, obsolete conversion thing from a previous minecraft version?

File: src/main/java/com/minecolonies/core/colony/buildings/modules/FarmerFieldModule.java
Comment (MotionlessTrain) [inline src/main/java/com/minecolonies/core/colony/buildings/modules/FarmerFieldModule.java:1]: No package update?

**Review by marchermans — DISMISSED** [review]

File: src/main/java/com/minecolonies/core/network/messages/server/colony/building/crusher/CrusherSetModeMessage.java
Comment (MotionlessTrain) [inline src/main/java/com/minecolonies/core/network/messages/server/colony/building/crusher/CrusherSetModeMessage.java:1]: Wouldn't the package have changed in this file?

File: src/main/java/com/minecolonies/core/tileentities/ScarecrowTileEntity.java
Comment (MotionlessTrain) [inline src/main/java/com/minecolonies/core/tileentities/ScarecrowTileEntity.java:1]: No package change?

**Review by MotionlessTrain — COMMENTED** [review]
I took the challenge to see whether I could review everything before midnight, and I succeeded!
I found a few things that looked a bit odd to me, mainly a small amount of files that didn't have changes where I would expect the package declaration line to have changed

**Comment by Thodor12** [comment]
@MotionlessTrain The 3 files that you found are actually empty files, they'd have to be removed

**Comment by MotionlessTrain** [comment]
> @MotionlessTrain The 3 files that you found are actually empty files, they'd have to be removed actually

That was the other option I thought of, but in that case I was wondering why there are empty java files in the project 😄

**Comment by Thodor12** [comment]
Well I know for sure one of them, the FarmerFieldModule, was the old field module which is no longer used, and I think that got back into the code through a merge request where one side thinks the file exists, the other wants it deleted, so when you merge that you can sometimes end up with the file created, but fully empty

**Comment by Thodor12** [comment]
I reckon that's what happened to all 3 of those files

File: src/api/java/com/minecolonies/api/colony/requestsystem/StandardFactoryController.java (hunk 1 above)
Comment (Raycoms) [reply src/api/java/com/minecolonies/api/colony/requestsystem/StandardFactoryController.java:322]: Old thing, yes.

**Pull request merged**